### DISCUSSION TITLE
Status of nuxt bridge

### DISCUSSION THREAD

**Discussion by AndreyYolkin** [nuxt/bridge#323] [General]
Hi, since nuxt/bridge was separated from main repo, it looks abandoned. In the same I rely on code I wrote for bridge version and I have some questions:

1. https://github.com/nuxt/bridge/blob/e7c71b50231c61ebd6bfb541dfe24d5a5df3c58c/src/runtime/app.ts#L76 May be better will be save state in temp variiable and restoring it instead of just deleting it? It causes 'Nuxt app is unavailable' in some code I wrote (I can't get constants from useNuxtApp() in setup() in some cases). Interesting is that it crashes only in production mode, dev works as expected.
2. Should we continue discussing nuxt/bridge here, because https://github.com/nuxt/bridge/discussions leads to 404?
3. What is the current status of nuxt/bridge? 18 days without updates for edge version is not cheerfully

EDIT: thank you for the fast reply

**Comment by danielroe** [comment]
Thanks for the ping. We now have Bridge discussions. Please feel free to create issues for any problems you're experiencing!
